src/api/operator/operatorApi.ts

```diff
@@ -157,18 +140,16 @@ export async function getTxOrders(params: GetTxOrdersParams): Promise<RawOrder[]
  *
  * Both filters cannot be used at the same time
  */
-export async function getTrades(params: GetTradesParams): Promise<RawTrade[]> {
-  const { networkId, owner = '', orderId = '' } = params
-  const cowInstance = COW_SDK[networkId]
-  const cowInstanceDev = COW_SDK_DEV[networkId]
-
-  if (!cowInstance || !cowInstanceDev) return []
-
+export async function getTrades(params: GetTradesParams & WithNetworkId): Promise<RawTrade[]> {
+  const { networkId, owner, orderId } = params
   console.log(`[getTrades] Fetching trades on network ${networkId} with filters`, { owner, orderId })
 
+  // sdk not merging array responses yet
   const trades = await Promise.all([
-    cowInstance.cowApi.getTrades({ owner, orderId }),
-    cowInstanceDev.cowApi.getTrades({ owner, orderId }),
+    // @ts-expect-error to avoid duplication we just pass both parameters
+    COW_SDK.cowApi.getTrades({ owner, orderId }, { chainId: networkId, env: 'prod' }),
+    // @ts-expect-error to avoid duplication we just pass both parameters
+    COW_SDK.cowApi.getTrades({ owner, orderId }, { chainId: networkId, env: 'staging' }),
```

Comment:
I don't get it, what's the TS error being suppressed?

Reply:
`owner` and `orderId` can't be passed together as both are mutual excluded.

Reply:
nit, would it help to do sth like 

```ts
const tradeParams = owner ? { owner } : { orderId }
```

It kind of suck to add two times the same hack, at least could be refactored so only one is needed. Anyways, maybe is not that bad to to try to comply with the TS rule